.github/workflows/ci.yaml

```diff
@@ -51,10 +51,10 @@ jobs:
       fail-fast: false
       matrix:
         juju:
-          - agent: 2.9.46
-            libjuju: ^2
+          - agent: 2.9.46  # renovate: latest juju 2
```

Comment:
The backup tests fail on 2.9.47. Charm cannot be redeployed with the same name.

Reply:
This seems to be [fixed](https://github.com/canonical/postgresql-operator/actions/runs/8615188370/job/23610463141?pr=433) with juju 2.9.49, but there are issues with the subordinates test. I will update in a separate PR.

Reply:
I cannot find such fixes mentioned in Juju 2.9.47+ release notes: https://discourse.charmhub.io/t/roadmap-releases/5064 
Maybe [this](https://bugs.launchpad.net/juju/+bug/2058506)... strange... Will ask Juju Team to improve release notes as 2.9.48 is missing there completely.